src/NzbDrone.Common/Extensions/UrlExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Common.Extensions
+{
+    public static class UrlExtensions
+    {
+        public static bool IsValidUrl(this string path)
+        {
+            if (string.IsNullOrWhiteSpace(path))
+                return false;
+
+            Uri uri;
+            if (!Uri.TryCreate(path, UriKind.RelativeOrAbsolute, out uri))
```

Comment:
I don't think it makes sense to allow relative URLs, unless sonnar would want to send notifications to itself. :smile: